custom/enikshay/tasks.py

```diff
@@ -382,17 +382,20 @@ def __init__(self, domain, episode_case):
         self.episode = episode_case
 
     @staticmethod
-    def _get_voucher_date(voucher):
+    def _get_fulfilled_voucher_date(voucher):
         return voucher.get_case_property('date_fulfilled')
 
-    def _get_vouchers(self):
-        all_vouchers = get_fulfilled_prescription_vouchers_from_episode(self.domain, self.episode.case_id)
+    @memoized
+    def _get_all_vouchers(self):
+        return get_prescription_vouchers_from_episode(self.domain, self.episode.case_id)
+
+    def _get_fulfilled_vouchers(self):
         relevant_vouchers = [
-            voucher for voucher in all_vouchers
+            voucher for voucher in self._get_all_vouchers()
             if (voucher.get_case_property('voucher_type') == 'prescription'
                 and voucher.get_case_property('state') == 'fulfilled')
         ]
-        return sorted(relevant_vouchers, key=self._get_voucher_date)
+        return sorted(relevant_vouchers, key=self._get_fulfilled_voucher_date)
```

Comment:
What's this about?  Isn't this the same logic as `get_fulfilled_prescription_vouchers_from_episode` except sorted?